Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java

```diff
@@ -236,6 +236,10 @@ public void reset() {
     public void storeCallBackMessageUpdated(final String channelId, final String uuid, final String callback) {
         storeChatMessageCallBack = callback;
         KeyValueStore.write(bridgeComponents.getContext(), bridgeComponents.getSdkName(), "CHAT_CHANNEL_ID", channelId); // Update the Local Storage Value
+         if(ChatService.chatChannelID != null && !ChatService.chatChannelID.equals(channelId) ) {
```

Comment:
what is the use case of this change?

Reply:
This is to restart the chat service whenever we encounter a different channel id even if we already started the chat service. So that we can restart the chat service when another channel id is encountered while switching the chat and restarting the chat service after killing the app.